runtime/rastrace/mt.tdf

```diff
@@ -23,7 +23,7 @@
 Executable=mt
 DATFileName=J9TraceFormat.dat
 
-TraceEntry=Trc_MethodEntry   Overhead=1 Level=5 Group=bytecodeMethods                Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx"
+TraceEntry=Trc_MethodEntry   Obsolete Overhead=1 Level=5 Group=bytecodeMethods       Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx"
```

Comment:
I think this tracepoint should stay (instead of using `Trc_MethodEntryStr` with a constant empty string).